samplelistener/listener/service_types.bal

```diff
@@ -1,10 +1,13 @@
-type SlackAppMentionHandlingService service object {
-   remote function onAppMention(GenericEventWrapperEvent e);
+# Service type to handle the events coming to the channel app/mention  
+public type AppMentionHandlingService service object {
+   remote function onAppMentionAdded(GenericEventWrapperEvent e);
+   remote function onAppMentionRemoved(GenericEventWrapperEvent e);
 };
 
-type SlackAppCreatedHandlingService service object {
+# Service type to handle the events coming to the channel app/create  
+public type AppCreatedHandlingService service object {
    remote function onAppCreated(GenericEventWrapperEvent e);
 };
 
-
-type genericType SlackAppMentionHandlingService | SlackAppCreatedHandlingService;
\ No newline at end of file
+# Generic service type for all the above service types  
+public type GenericService AppMentionHandlingService | AppCreatedHandlingService;
```

Comment:
We don't usually add spaces around | in a union type.
```suggestion
public type GenericService AppMentionHandlingService|AppCreatedHandlingService;
```

Reply:
Resolved